rosmon_core/src/main.cpp

```diff
@@ -372,37 +378,41 @@ int main(int argc, char** argv)
 					fmtNoThrow::print(stderr, "Could not create rosmon/core_dumps directory\n");
 				}
 
-				logDir = workDir + "/roslogs";
-				if (chdir(logDir.c_str()) == 0 || mkdir(logDir.c_str(), 0777) == 0) 
-				{
-					logFile = logDir + "/" + launchInfo.launch_group + "_" + launchInfo.launch_config + ".log";
-				}
-				else
-				{
-					fmtNoThrow::print(stderr, "Could not create rosmon/roslogs directory\n");
+				if (!disableLog) {
```

Comment:
It seems like in retrospect, would have made more sense to define a `enableLog` variable instead to avoid these double negations everywhere

Reply:
when we do mon launch via terminal we want it to be enable logs by default, so this is better